scripts/build_mpc_c_test.zsh

```diff
@@ -110,9 +110,9 @@ mpc_test 2 ./examples/C/mpc/unit_tests/misc_tests/2pc_millionaires.c
 mpc_test 2 ./examples/C/mpc/unit_tests/misc_tests/2pc_multi_var.c
 
 # build hycc benchmarks
-mpc_test 2 ./examples/C/mpc/benchmarks/biomatch/2pc_biomatch.c
-mpc_test 2 ./examples/C/mpc/benchmarks/kmeans/2pc_kmeans.c
-mpc_test 2 ./examples/C/mpc/benchmarks/kmeans/2pc_kmeans_og.c
+# mpc_test 2 ./examples/C/mpc/benchmarks/biomatch/2pc_biomatch.c
+# mpc_test 2 ./examples/C/mpc/benchmarks/kmeans/2pc_kmeans.c
+# mpc_test 2 ./examples/C/mpc/benchmarks/kmeans/2pc_kmeans_og.c
```

Comment:
Just to confirm: Why you made this change?

Reply:
Made this change because 0 cost operations (from Select and Store) in the ILP solver will sometimes cause an infinite loop. 

I'll adjust the commit message, panic on Select and Store in the ILP solver, and comment out all Array tests. I'll uncomment these tests when I merge in the def-use-graph changes.